xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -218,14 +197,7 @@
       <Docs>
         <summary>Enables the <see cref="T:System.Windows.Forms.SplitterPanel" /> to shrink when <see cref="P:System.Windows.Forms.SplitterPanel.AutoSize" /> is <see langword="true" />. This property is not relevant to this class.</summary>
         <value>One of the <see cref="T:System.Windows.Forms.AutoSizeMode" /> values. The default is <see cref="F:System.Windows.Forms.AutoSizeMode.GrowOnly" />.</value>
```

Comment:
@rpetrusha what if this property is relevant to other classes? Then maybe it's good to make sure we mention the default. It doesn't hurt anyway.

Reply:
SplitterPanel is sealed, so it can't be relevant to other classes. I think the text was copied from a control for which the property value is meaningful..